src/meltano/core/tracking/tracker.py

```diff
@@ -427,3 +430,25 @@ def track_block_event(self, block_type: str, event: BlockEvents) -> None:
                 {"type": block_type, "event": event.name},
             )
         )
+
+    def track_exit_event(self):
+        from meltano.cli import exit_code
+
+        start_time = datetime.utcfromtimestamp(Process().create_time())
+
+        # This is the reported "end time" for this process, though in reality the process will end
+        # a short time after this time as it takes time to emit the event.
+        now = datetime.utcnow()
+
+        self.track_unstruct_event(
+            SelfDescribingJson(
+                "iglu:com.meltano/exit_event/jsonschema/1-0-0",
```

Comment:
Similar to other examples at top of this file, I think breaking this into two constants is a good idea - one for the schema path and one for the version number.

Reply:
@aaronsteers How do you feel about me centralizing all of these schema-version pairs of constants into a dictionary in `src/meltano/core/tracking/schema.py` that maps from schema name to schema version with a function to get the full URL. That way we won't end up with any duplicates or version mismatches.

Reply:
I'm not sure. Do you mean, we'd have the constants for the version string, but the lookup to version would be by passing in the constant?

Honestly, I think might be better to safe for another iteration so we can discuss more.

Reply:
I'll make a follow-up PR to illustrate my suggestion.

Reply:
https://github.com/meltano/meltano/pull/6146